sonar-project.properties

```diff
@@ -0,0 +1,7 @@
+sonar.organization=the-virtual-brain
+sonar.projectKey=the-virtual-brain_tvb-ext-unicore
+
+sonar.python.version=3.8, 3.9
```

Comment:
Could this work also on Python 3.10 ?

Reply:
I added it and the sonar scan is passing, the result is the same. I am curious whether Sonar would indicate whether a code suggestion/vulnerability is part of a certain version of python or would just display it generically.